namespaces/cloud-platform-live-0.k8s.integration.dsd.io/family-mediators-api-staging/resources/rds.tf

```diff
@@ -0,0 +1,29 @@
+############################################
+# Family Mediators API RDS (postgres engine)
+############################################
+
+module "rds-instance" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=2.1"
```

Comment:
ref=3.0

Reply:
Yep, I will upgrade this to release 3.0

Reply:
yes, was just making sure - approved

Reply:
np, a small edit shouldn't require another approval